ui-grid/ui-grid.d.ts

```diff
@@ -3766,6 +3766,14 @@ declare module uiGrid {
          * You may specify one of the sortingAlgorithms found in the rowSorter service.
          */
         sortCellFiltered?: boolean;
+        /**
+         *(optional) An array of sort directions, specifying the order that they should cycle through as 
```

Comment:
I'd also remove (optional) since that is already indicated by the `?`
to add in the specific types you do:
```
sortDirectionCycle?: Array<IUiGridConstants.ASC | IUiGridConstants.DESC>
```